administrator/language/de-DE/mod_toolbar.ini

```
@@ -5,4 +5,6 @@
 ; Note : All ini files need to be saved as UTF-8
 
 MOD_TOOLBAR="Werkzeugleiste"
+MOD_TOOLBAR_FIELD_TOOLBAR_LABEL="Werkzeugleiste-Kennung"
+MOD_TOOLBAR_FIELD_TOOLBAR_DESCRIPTION="Die Standard-Werkzeugleiste im Administrator heißt 'Werkzeugleiste'. Wenn eine Erweiterung eine eigene Werkzeugleiste erzeugt, kann ein eigenes Modul mit einer eigenen Werkzeugleisten-Kennung erstellt werden. Die Standard-Werkzeugleiste sollte niemals entfernt werden!"
```

```suggestion
MOD_TOOLBAR_FIELD_TOOLBAR_DESCRIPTION="Die Standard-Werkzeugleiste im Administrator heißt 'toolbar'. Wenn eine Erweiterung eine eigene Werkzeugleiste erzeugt, kann ein eigenes Modul mit einer eigenen Werkzeugleisten-Kennung erstellt werden. Die Standard-Werkzeugleiste sollte niemals entfernt werden!"
```
sollte nicht übersetzt werden

---

Deswegen ja noch wip, konnte es noch nicht testen.

---

Kein Problem